tests/test_resolve_structures.py

```
@@ -0,0 +1,72 @@
+def test_resolve_structures_dict_empty_indices(water_frame):
+    """Dict with empty index list produces empty molecules and masses."""
+    structures = {"empty": []}
+    molecules, masses = resolve_structures(water_frame, structures)
+    assert molecules["empty"] == ()
+    assert masses["empty"] == []
+
+
+def test_resolve_structures_list_no_match():
+    """List with SMILES that don't match produces no entries for that species."""
+    atoms = ase.Atoms("Ar", positions=[[0, 0, 0]])
+    molecules, masses = resolve_structures(atoms, ["[Li+]"])
```

Ruff (0.15.6)

[warning] Unpacked variable `masses` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)